feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/billingClientParamBuilders.kt

```diff
@@ -0,0 +1,12 @@
+package com.revenuecat.purchases.google
+
+import com.android.billingclient.api.QueryPurchaseHistoryParams
+import com.android.billingclient.api.QueryPurchasesParams
+
+fun String.buildQueryPurchaseHistoryParams(): QueryPurchaseHistoryParams {
+    return QueryPurchaseHistoryParams.newBuilder().setProductType(this).build()
```

Comment:
if we check that the String is a BillingClient.ProductType then we have to make the return type nullable and I'm not sure that makes it worth it. thoughts @vegaro ?

Reply:
Personally I think that makes sense to guard against possible issues... Otherwise, in case this is passed a wrong string, would it crash? I couldn't find the behavior in that case in the docs. I think we would want crashing or failing silently, I think we should log an error if this happens.

Reply:
I think you can also annotate the type of the receiver right? I think it's something like `@receiver:BillingClient.ProductType`. In any case that would just print a warning in case we are calling this function with the wrong stuff.

Reply:
Ohh I didn't know about that but it's nice!